lambdas/process-journey-event/src/test/java/uk/gov/di/ipv/core/processjourneyevent/NestedJourneyTypes.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.di.ipv.core.processjourneyevent;
```

Comment:
this file lives in `uk.gov.di.ipv.core.processjourneyevent` instead of `uk.gov.di.ipv.core.processjourneyevent.statemachine` as otherwise it would overload the "real" nested journey type definitions

Reply:
Do you need it to be an enum for the tests - could it just be a `List<String>`?

Reply:
good point! 🤦🏽‍♀️